app/views/posts/_expanded.html.erb

```diff
@@ -447,8 +448,10 @@
                 <% elsif post.comments_disabled %>
                   <p class="has-color-red-500"><i class="fa fa-lock"></i> Comments have been disabled on this post, but as a moderator you are exempt from that block.</p>
                 <% end %>
-                <a href="#" class="button is-muted is-small" data-modal=".js--<%= post.id %>-new-thread-modal">Start new comment thread</a>
+<!--                <a href="#" class="button is-muted is-small" data-modal=".js--<%#= post.id %>-new-thread-modal">Start new comment thread</a>-->
+                <a class="button is-muted is-small" onclick="new_thread(<%= post.id %>)">Start new comment thread</a>
```

Comment:
Related to my other comment about not adding to the `window` scope: instead of using an `onclick` attribute on this element, add a class to it such as `js-new-thread-link`, and add a click handler in the JS file, such as with `$(document).on('click', '.js-new-thread-link', ev => { });`

Remember that you can add `data-*` attributes to this element if you need parameters. Something like this would work:

```suggestion
                <a class="button is-muted is-small js-new-thread-link" data-post="<%= post.id %>">Start new comment thread</a>
```

This is a common pattern in our JS, so you can find examples if you need it. Your JS will probably look something like this:

```js
$(document).on('click', '.js-new-thread-link', ev => {
  const $tgt = $(ev.target);
  const postId = $tgt.attr('data-post');
  // carry on from here as your existing function does
});
```